extensions/eo/examples/example-landsat8.json

```diff
@@ -48,8 +48,6 @@
         "landsat:path": 107,
         "landsat:row": 18,
         "eo:gsd": 30,
-        "eo:platform": "landsat-8",
-        "eo:instrument": "oli_tirs",
         "eo:off_nadir": 0,
```

Comment:
I would prefer to see these updated to use the `instrument` fields, since we already have other fields that are not eo (e.g., the non-standard `landsat:path` field)